<launch>
 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="iroad_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="accel_id" value="5" type="int"/>
  <param name="steer_id" value="0" type="int"/>
  <param name="ct_input_id" value="2" type="int"/>
  <param name="PRNDL_vr_id" value="1" type="int"/>
  <param name="PRNDL_ct_id" value="2" type="int"/>
  <param name="pk_brake_id" value="0" type="int"/>
  <param name="cl_act_id" value="3" type="int"/>
  <param name="steer_scale" value="350" type="double"/>
  <param name="lat0" value="37.4299484" type="double"/>
  <param name="lon0" value="-122.1836507" type="double"/>
  <param name="ptx" value="-31.0" type="double"/>
  <param name="pty" value="5.0" type="double"/>
  <param name="dthres" value="5.0" type="double"/>
  <node pkg="bridge_iroad" type="iroad_closedloop" name="iroad_closedloop" output="screen"/>
</launch>
